src/App.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import { Routes, Route, Link } from 'react-router-dom';
+import Books from './components/Books';
+import Categories from './components/Categories';
+import './App.css';
+
+function App() {
+  const [books, setBooks] = useState([]);
+  const handleAddBook = (newbook) => {
+    setBooks((books) => [...books, newbook]);
+  };
+
+  const handleRemovebook = (bookid) => {
+    setBooks((books) => books.filter((book) => book.id !== bookid));
+  };
+  return (
+    <>
+      <nav>
+        <div>
+          <h1> Book Store</h1>
+        </div>
+        <div>
+          <ul>
+            <li>
+              <Link to="/">Books</Link>
+            </li>
+            <li>
+              <Link to="/categories">Categories</Link>
+            </li>
+          </ul>
+        </div>
+      </nav>
```

Comment:
- [Optional] Good job so far, I can suggest you create a new component and add the Navbar inside then you can just import the `Navbar` function here in `App`. It's a good approach to have each component in its file so if on have a problem you can debug without any problem or without touching other files 🙃